README.md

```diff
@@ -41,6 +41,16 @@ Source-build solves common challenges that most developers encounter when trying
 
 Starting with .NET 6, the core source-build infrastructure is integrated into the [dotnet/installer](https://github.com/dotnet/installer/tree/main/src/SourceBuild) repo. The `main` branch on this repo now contains the tooling needed to build .NET's external dependencies from source.
 
+## Linux distributions building and offering .NET
+| Distro |.NET Packages|Maintainer|Feed|Preview/Beta Packages|
```

Comment:
What do you think about the following:

1. List the github alias of the actual maintainers.
- Alpine: ayakael
- ArchLinux: alucryd
- CentOS: omajid
- Fedora: omajid, crummel
- Homebrew (macOS): asbjornu
- RHEL: omajid
- Ubuntu: mirespace
2. The text in the packages column is not adding unique value.  What about combining the feed and packages columns?

Reply:
> What do you think about the following:
> 
> 1. List the github alias of the actual maintainers.
> 
> * Alpine: ayakael
> * ArchLinux: alucryd
> * CentOS: omajid
> * Fedora: omajid, crummel
> * Homebrew (macOS): asbjornu
> * RHEL: omajid
> * Ubuntu: mirespace
> 
> 2. The text in the packages column is not adding unique value.  What about combining the feed and packages columns?

1. This will require us to keep watching and updating these names which are subject to change.

Reply:
I don't see that as burdensome.  The ROI is worthwhile IMO.

Reply:
Added